src/cascadia/TerminalApp/GlobalAppSettings.h

```diff
@@ -42,6 +42,8 @@ class TerminalApp::GlobalAppSettings final
     bool GetShowTitleInTitlebar() const noexcept;
     void SetShowTitleInTitlebar(const bool showTitleInTitlebar) noexcept;
 
+	void SetRequestedTheme(const winrt::Windows::UI::Xaml::ElementTheme requestedTheme) noexcept;

```

Comment:
You used tabs, whereas the convention is 4 spaces :)

Reply:
ops!